### DISCUSSION TITLE
Should I make WallPanel step-by-step instructions?

### DISCUSSION THREAD

**Discussion by mngarchow** [dinki/View-Assist#9] [General]
Hi, just wondering if you want me to make step-by-step written instructions for installing WallPanel? I'm happy to do so, just want to make sure you haven't already started them.

**Comment by dinki** [comment]
I can't tell you how much I appreciate your help with this and will not turn you down on this or any other. This will definitely help me have more time to record more videos. 

Thanks again. You are a champion
